services/welding-machine-defect-detection-model-service/app/services/predict_service.py

```
@@ -0,0 +1,50 @@
+import numpy as np
+from app.models.model_loader import load_model_file, load_scaler, load_threshold
```

Ruff (0.12.2)

`app.models.model_loader.load_model_file` imported but unused

Remove unused import

(F401)